.github/workflows/go-releaser.yaml

```diff
@@ -23,4 +23,5 @@ jobs:
           version: latest
           args: release --rm-dist
         env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@mrchrisadams Another fix is needed. I thought this env var was added automatically but the action fails with

```
 ⨯release failed after 0.00serror=missing GITHUB_TOKEN, GITLAB_TOKEN and GITEA_TOKEN
Error: The process '/opt/hostedtoolcache/goreleaser-action/1.9.2/x64/goreleaser' failed with exit code 1
```